rotkehlchen/chain/evm/decoding/thegraph/balances.py

```diff
@@ -195,7 +146,10 @@ def query_balances(self) -> 'BalancesSheetType':
                 continue
             balance = FVal(shares_amount * pool_total_tokens / pool_total_shares)
             balance_norm = asset_normalized_value(balance.to_int(exact=False), self.token)
-            balances[delegator].assets[self.token] += Balance(
+            user = ThegraphCommonDecoder.get_user_address(self.evm_inquirer, delegator)
+            if user is None:
+                continue
```

Comment:
what is user and delegator here? is delegator the address of the vested contract? shouldn't you check before anything that youi have it right and user here is tracked? Before querying anything since it could be that there was an error in the decoding logic

Reply:
Simplified this to just use event.location_label as user_address, as we already have that checked.